src/OpenTelemetry.Exporter.Instana/Implementation/Transport.cs

```diff
@@ -26,8 +26,9 @@ namespace OpenTelemetry.Exporter.Instana.Implementation;
 
 internal class Transport
 {
-    private const int MultiSpanBufferLimit = 4096000;
-
+    private const int MULTI_SPAN_BUFFER_SIZE = 4096000;
+    private const int MULTI_SPAN_BUFFER_LIMIT = 4070000;
+    private readonly byte[] tracesBuffer = new byte[MULTI_SPAN_BUFFER_SIZE];
```

Comment:
Please fix compilation errors.
(It should be easily reproducible locally if you switch Debug to Release).

Reply:
👍